includes/class.llms.person.handler.php

```diff
@@ -964,7 +964,14 @@ public static function validate_fields( $data, $screen = 'registration' ) {
 			// Check email field for uniqueness.
 			if ( 'email_address' === $name ) {
 
-				$skip_email = false;
+				/**
+				 * Allow for skipping of the emxail_exists check.
+				 *
+				 * @since x.x.x
+				 *
+				 * @param bool $skip_email A value of true will skip checking if the email exists.
+				 */
+				$skip_email = apply_filters( 'llms_skip_email', false );
```

Comment:
As submitted I feel the hook name is too generic, something like this feels better to me:

```suggestion
				$skip_email = apply_filters( 'llms_skip_new_user_email_dupcheck', false );
```